src/elfpy/strategies/single_lp.py

```diff
@@ -6,33 +6,29 @@
 # pylint: disable=duplicate-code
 # pylint: disable=too-many-arguments
 
+import logging
+
 from elfpy.strategies.basic import BasicPolicy
 
 
 class Policy(BasicPolicy):
-    """
-    simple LP
-    only has one LP open at a time
-    """
-
-    def __init__(self, market, rng, wallet_address, budget=1000):
-        """call basic policy init then add custom stuff"""
-        self.amount_to_lp = 100
-        super().__init__(
-            market=market,
-            rng=rng,
-            wallet_address=wallet_address,
-            budget=budget,
-        )
+    "single LP: only has one LP open at a time"
```

Comment:
This is now a static element that belongs to the class. Before, the variable was an object member variable (belongs to `self`). The difference is whether you want the variable to exist even if the class is not instantiated. We want `amount_to_lp` to be owned by some instantiated user. e.g.
```
from elfpy.strategies.single_lp import Policy
alice = Policy()
alice.amount_to_lp = 100
bob = Policy()
bob.amount_to_lp = 500
```

so in this example `amount_to_lp` belongs to alice or bob, not to Policy.  As you have it above, it belongs to the policy class.

tl;dr -- put it back in the init

Reply:
ah that makes the difference clear, thanks. i was mostly trying to remove __init__ for brevity, which again results in screwing things up 🤦‍♂️